docs/maintainers/adr/0020-reduce-support-for-internet-explorer.md

```diff
@@ -0,0 +1,77 @@
+# Reduce support for Internet Explorer
+
+* Status: accepted
+* Date: 2023-12-19
+
+Technical Story: PLATUI-2686
+
+## Context and Problem Statement
+
+[govuk-frontend v5.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.0.0)
+drops support for Internet Explorer (IE) versions 8 through 10 by preventing loading of its Javascript (JS)
+and removing various IE-specific polyfills.  IE11 is still supported via graceful degradation to a non-JS experience.
+
+We propose to follow the same approach in our frontend libraries, preventing loading of our own JS and
+removing any IE-specific polyfills.
+
+## Decision Drivers
+
+* Direction taken by Government Digital Service
+* Industry trends to drop support for IE
+* Overhead of maintaining and testing IE-specific implementations
+* Minimal number of users still using IE (recent sample found the number was less than 0.6% of users over a 2-week period)
+* Security risk to end-users of continuing to use a browser beyond its End-Of-Life date
+* Impact to service teams using our libraries
+
+## Considered Options
+
+* Drop support for IE entirely in our libraries
+* Provide minimal support for IE in our libraries
+* Continue supporting IE in our libraries
+
+## Decision Outcome
+
+Chosen option: "Provide minimal support for IE in our libraries" because, whilst we don't want to continue supporting
```

Comment:
Again, I think it might be helpful to define `our libraries` for future maintainers, as this relates to both this repo but also `hmrc-frontend` (but not `govuk-frontend`).

Reply:
Good shout - I've made this explicit where they're first mentioned